dietpi/dietpi-software

```diff
@@ -6928,6 +6936,17 @@ exec sudo -u $ha_user dash -c '$ha_pyenv_activation; exec pip3 install -U homeas
 			fi
 
 		fi
+    
+    software_id=137 # Unbound
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 ))
+		then
+			Banner_Installing
+
+      G_AGI unbound
+      
+      G_DIETPI-NOTIFY It is possible that Unbound just gave an error. This is safe to ignore.
```

Comment:
`G_DIETPI-NOTIFY` requires a numerical first argument:
- `G_DIETPI-NOTIFY 2 Info message`
- `G_DIETPI-NOTIFY 1 Error message`
- `G_DIETPI-NOTIFY 0 Ok message`
- `G_DIETPI-NOTIFY 3 Header message`
- `G_DIETPI-NOTIFY -2 Processing animation`

Info message looks reasonable here. It is okay to split the message across multiple arguments, however to minimise processing time and preserve any format, it is advised to quote it:
```
G_DIETPI-NOTIFY  2 'It is possible that Unbound just gave an error. This is safe to ignore.'
```